docs/src/Callout.doc.js

```diff
@@ -22,168 +47,354 @@ card(
+  </MainSection>,
 );
 
 card(
-  <Example
-    name="Error Example"
-    defaultCode={`
+  <MainSection name="Variants">
+    <MainSection.Subsection
+      title="info"
+      description="Info Callouts communicate helpful messages to users about the product. In most cases, they should provide an action for users to take."
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+<Callout
+  type="info"
+  iconAccessibilityLabel="Info"
+  title="Your business account was created!"
+  message="Apply to the Verified Merchant Program"
+  primaryAction={{href: "https://pinterest.com", label: "Get started", accessibilityLabel:"Get started: Verified Merchant Program"}}
+  secondaryAction={{href: "https://pinterest.com", label: "Learn more", accessibilityLabel:"Learn more: Verified Merchant Program"}}
+  dismissButton={{
+    accessibilityLabel: 'Dismiss this banner',
+    onDismiss: ()=>{},
+  }}
+/>
+`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Warning"
+      description="Warning Callouts communicate cautionary messages to users. Action should not be required. The Callout should provide clear guidance on how to correct an issue and/or learn more about it."
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+<Callout
+  type="warning"
+  iconAccessibilityLabel="Warning"
+  title="Unused ActAlike audiences are going away"
+  message="We have noticed that you have ActAlike audiences in your advertiser account that have been used in an ad campaign. Pinterest will be deleting any unused ActAlike audiences on May 30, 2020."
+  primaryAction={{href: "https://pinterest.com", label:"View audiences"}}
+  dismissButton={{
+    accessibilityLabel: 'Dismiss warning',
+    onDismiss: ()=>{},
+  }}
+/>
+`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Error"
+      description="Error Callouts inform users of problems that require immediate action to correct. Further actions on the page might be blocked if users do not correct the problems. The Callout should also provide clear guidance on how to correct the issue and/or learn more about it."
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
 <Callout
   type="error"
-  iconAccessibilityLabel="Error icon"
-  message="This action can't be undone."
+  title="Pinterest tag needs attention"
+  iconAccessibilityLabel="Error"
+  message="Your tag has errors, so information may be outdated. Fix your tag for the most accurate metrics."
+  primaryAction={{href: "https://pinterest.com", label:"Fix tag"}}
+
 />
-  `}
-  />,
-);
+`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Actions"
+      description={`
+        Callouts can have either one primary action, or a secondary action and a primary action. These actions can be [Links](/Link), by specifying the \`href\` property, or [Buttons](/Buttons), when no \`href\` is supplied.
```

Comment:
not strong about this, but 

        `Callouts can have either one primary action, or a _primary_ action and a _secondary_ action.`